### PR TITLE
Added 3 new pages, AWG, EWG and how to join them

### PR DESCRIPTION
**Pull request by linkfang**
Signed-off-by: Zhou Fang <zhou.fang@eclipse-foundation.org>

Fixes https://github.com/openhwgroup/openhwgroup.org/issues/434

### CODE REVIEW

**Comment by linkfang** [comment]
Got +1 from Duncan, https://github.com/openhwgroup/openhwgroup.org/issues/434#issuecomment-1113626289

**Review by autumnfound — CHANGES_REQUESTED** [review]
Do not merge. See bottom of working-groups page, it is including list entries for the other pages which is not what we want. You'll want to change the layout of the working groups page to account for this (I believe we want single here)

**Comment by linkfang** [comment]
Got +1 from Duncan for updated content, https://github.com/openhwgroup/openhwgroup.org/issues/434#issuecomment-1117666909

**Review by autumnfound — APPROVED** [review]

**Pull request merged**
